ansible/templates/usb/scripts/prompt-config.sh.j2

```diff
@@ -532,17 +550,19 @@ promptnic()
 	printnics
 	num=0
 	while [ /usr/bin/true ]; do
-		prmpt_str="Enter the number of the NIC for the $1 interface: "
+		prmpt_str="Enter the number of the NIC for the $tag interface: "
 		printf "$prmpt_str"
 		read num
 		if ! [[ "$num" =~ ^[0-9]+$ ]] ; then
 				echo ""
 		elif [ $num -ge 1 -a $num -le $nic_cnt ]; then
```

Comment:
Please use double brackets for consistency and then you can use `&&` instead of `-a`, which improves code readability a lot (`-a` vs `&&`)